src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -265,26 +220,14 @@ export const MultiSelect = ({
                                 />
                             )}
 
-                            {!filterable && activeItemKeys.length === 0 && placeholder && (
-                                <Text color="weak">{placeholder}</Text>
-                            )}
+                            {activeItemKeys.length === 0 && placeholder && <Text color="weak">{placeholder}</Text>}
                         </div>
-                        {filterable && (
-                            <input
-                                ref={filterInputRef}
-                                className="focus:tw-outline-0 tw-text-s tw-bg-transparent"
-                                style={{ maxWidth: inputWidth }}
-                                placeholder={activeItemKeys.length === 0 ? placeholder : filterLabel}
-                                onChange={handleFilterChange}
-                            />
-                        )}
                     </div>
                 </div>
             </Trigger>
 
             {open &&
                 heightIsReady &&
-                emphasis === MultiSelectEmphasis.Default &&
                 createPortal(
```

Comment:
We can use the new `Portal` component to wrap this div.

Reply:
Done